persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java

```diff
@@ -41,6 +41,8 @@ private PersistToolsConstants() {}
     public static final String KEY_PASSWORD = "password";
     public static final String KEY_DATABASE = "database";
     public static final String KEY_HOST = "host";
+    public static final String KEYWORD_PACKAGE = "package";
+    public static final String KEYWORD_NAME = "name";
```

Comment:
There should be constants for this in ballerina lang. Lets use that.

Reply:
> Lets check the same flow in init command too. Here if the directory is emoty we have to create file with same name as the package isnt it? We can improve that too.
this was updated in [5b06764](https://github.com/ballerina-platform/persist-tools/pull/127/commits/5b06764e6b339d0cd379fd6a908b8c2ec32a7107). For the constant from lang side, I couldn't find it in the `io.ballerina.runtime.api.constants` and `org.wso2.ballerinalang.compiler.util` packages. Is there any particular place to look?

Reply:
Have you checked in PackageConstants?

Reply:
This change will be done in a later update.